org.adempiere.base/src/org/compiere/print/DataEngine.java

```diff
@@ -1222,15 +1242,26 @@ else if (m_group.isFunctionColumn(pdc.getAD_PrintFormatItem_ID(), functions[f]))
 							name = Msg.getMsg(format.getLanguage(), PrintDataFunction.getFunctionName(functions[f]));
 						else
 							name = PrintDataFunction.getFunctionSymbol(functions[f]);	//	Symbol
+ 						if (m_group.isFunctionColumn(pdc.getAD_PrintFormatItem_ID(), functions[f]))
+						{
+ 							Serializable groupValue = m_group.getValue(PrintDataGroup.TOTAL, 
+									pdc.getAD_PrintFormatItem_ID(), functions[f]);
+							if(groupValue instanceof Timestamp)
+								groupValue = DisplayType.getDateFormat(pdc.getDisplayType()).format((Timestamp)groupValue);
+							if(groupValue != null)
+								name += ": " + groupValue;
+						}
```

Comment:
why special treatment for timestamp only ? wouldn't that create inconsistency here as it is only symbol here for other data type ?